plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakePerTransactionMetastoreCache.java

```diff
@@ -98,6 +98,7 @@ private DistributedQueryRunner createQueryRunner(boolean enablePerTransactionHiv
         DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(session).build();
 
         queryRunner.installPlugin(new TestingDeltaLakePlugin(
+                Optional.empty(),
```

Comment:
as a followup, this test class can perhaps be simplified with the `CountingAccess{File}HiveMetastore` class.
i.e. we could remove reflection use from here.

for that, we would need to ensure every method in `CountingAccess{File}HiveMetastore` is counted, not only selected ones.